### ISSUE TITLE
Upgrade version of nodejs

### ISSUE THREAD

**Issue by tschuy** [osuosl/timesync#20]
I need some ES6 in my life.

Currently we're using 0.10 so that we can run this in production easily. Let's try to find an easy to install 0.12 version for Centos!

**Comment by tschuy** [comment]
```
$ curl -O https://nodejs.org/dist/v0.12.6/node-v0.12.6-linux-x64.tar.gz
$ tar -xzf node-v0.12.6-linux-x64.tar.gz
$ node-v0.12.6-linux-x64/bin/node --version
v0.12.6
```

**Comment by tschuy** [comment]
Fixed by #21

**Comment by iankronquist** [comment]
Fixed by #21 and the [PR to the workstation cookbook](https://github.com/osuosl-cookbooks/workstation/pull/36). Both were merged.

**Comment by tschuy** [comment]
Did not mean to merge #21 . That was a mistake. But it's in now, and it works. /me shrugs
